packages/admin/src/cli.ts

```diff
@@ -20,27 +20,32 @@ const COMMON_COMMAND_ARGUMENTS = {
     mnemonic: {
       type: 'string',
       demandOption: true,
-      describe: 'Mnemonic phrase for the Airnode master wallet',
+      describe: 'Mnemonic phrase for the Airnode wallet',
     },
     derivationPath: {
       type: 'string',
       describe: 'Derivation path to be used for deriving the wallet account',
     },
   },
-  requester: {
+  sponsor: {
     type: 'string',
     demandOption: true,
-    describe: 'Requester address',
+    describe: 'Sponsor address',
   },
-  airnodeId: {
+  airnode: {
     type: 'string',
     demandOption: true,
-    describe: 'The onchain ID of the Airnode instance',
+    describe: 'Address of the Airnode wallet',
   },
-  clientAddress: {
+  sponsorWallet: {
+    type: 'string',
+    demandOption: true,
+    describe: 'Address of the Sponsor wallet',
```

Comment:
I see that some of the words are capitalized (Sponsor, Airnode) an some are not (requester). It should be somehow consistent I think.

Reply:
My bad. I'll make this naming consistent.